cmd/vc-rest/startcmd/start.go

```diff
@@ -95,6 +96,10 @@ func startEdgeService(parameters *vcRestParameters) error {
 		return errMissingHostURL
 	}
 
+	if parameters.edvURL == "" {
```

Comment:
You can add a unit test for this. Call it TestStartCmdWithBlankEDVArg()

Reply:
done

Reply:
done